### PR TITLE
[select] fix: set aria-disabled attribute on combobox elements

### PR DESCRIPTION
**Pull request by bvandrc**
Disabled prop should be applied to the element that has `role="combobox"`, not just the `input`.

### CODE REVIEW

**Comment by adidahiya** [comment]
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled#overview):

> The disabled attribute is supported by [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button), [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset), [`<keygen>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/keygen), [`<optgroup>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup), [`<option>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option), [`<select>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select), [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) and [`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).

So it sounds like it's not valid on `<div>` or `<span>` elements... do you have any documentation to support this change?

**Comment by bvandrc** [comment]
> According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled#overview):
> 
> > The disabled attribute is supported by [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button), [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset), [`<keygen>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/keygen), [`<optgroup>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup), [`<option>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option), [`<select>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select), [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) and [`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
> 
> So it sounds like it's not valid on `<div>` or `<span>` elements... do you have any documentation to support this change?

Ah, you're right. I thought changing via `role="combobox"` would allow `disabled`. But this documentation says:

image

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

Switched to `aria-disabled`.

File: packages/select/src/components/multi-select/multiSelect2.tsx [hunk 1]
```diff
@@ -289,6 +289,7 @@ export class MultiSelect2<T> extends AbstractPureComponent2<MultiSelect2Props<T>
                     ...popoverTargetProps,
                     ...targetProps,
                     "aria-expanded": isOpen,
+                    "aria-disabled": disabled,
```
Comment (adidahiya) [inline packages/select/src/components/multi-select/multiSelect2.tsx]: you need to swap these lines (yes, I know the lint rule is kind of annoying)

Comment (bvandrc) [reply packages/select/src/components/multi-select/multiSelect2.tsx]: lint rules are there for good reason! Fixed.

**Review by adidahiya — APPROVED** [review]

**Pull request merged**